client/src/components/Post/Post.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+.dots {
+  padding-bottom: 0;
+  font-weight: bold;
+  font-size: 1.5em;
+  line-height: 3px;
+  cursor: default;
+}
+
+.navig {
+  position: absolute;
+  left: -50vw;
```

Comment:
what the effect it give u?

Reply:
If you are talking about cursor it makes it look as if it was button.